app/models/base_channel_details.rb

```diff
@@ -1,4 +1,4 @@
-class BaseChannelDetails < ActiveRecord::Base
+class BaseChannelDetails < ApplicationRecord
```

Comment:
This one caused a headache as Rails changed the way connects_to works. In the tests, it was using a separate connection which couldn't see the changes in the transaction for the site_channels_controller test. Since it shares the same DB preferences as the application_controller, I just subclassed it.